src/main/resources/static/css/style.css

```diff
@@ -2944,32 +2944,23 @@ input:checked+.form-slider:before {
 	display: none;
 }
 
-.animate
-{
+.animate {
 	transition: all 0.1s;
-	-webkit-transition: all 0.1s;
 }
 
-.action-button
-{
-	position: relative;
+.action-button {
 	padding: 5px 10px;
 	margin: 0px 5px 5px 0px;
-	float: right;
 	border-radius: 5px;
-	color: #FFF;
-	text-decoration: none;
+	color: white;
 }
```

Comment:
Note: just found Schámán's `.button-holder` class in this style.css, you should add these properties to this `.action-button` class as well:
```css
text-transform: uppercase;
font-weight: 900;
```

Reply:
Also this:
```css
cursor: pointer;
```